src/site/resources/js/redirect.js

```diff
@@ -0,0 +1,14 @@
+window.onload = () => {
+    const currentUrl = window.location.href;
+    switch (currentUrl) {
+        case "https://checkstyle.org/checks/":
+            window.location.href = "https://checkstyle.org/checks";
+            break;
+        case "https://checkstyle.org/config_coding.html":
+            window.location.href = "https://checkstyle.org/checks/coding/";
```

Comment:
Please make matching or currentUrl to match  without domain 
To make link https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/eac018a_2024165644/config_coding.html to work

Reply:
I have pushed the new code, this new code isn't working properly locally, I have just pushed it to see if it works by any chance on github or not.